index.html

```diff
@@ -310,23 +314,17 @@ <h3>The <dfn><a>PerformanceMark</a> Constructor</dfn></h3>
             Set <var>entry</var>'s <code>startTime</code> attribute as follows:
             <ol>
               <li>
-                If <var>markOptions</var> is present and its <a>startTime</a> member is present, then:
+                If <var>markOptions</var>'s <a>startTime</a> member is <a data-cite="WEBIDL#dfn-present">present</a>, then:
                 <ol>
-                  <li>If <var>markOptions</var>'s <a>startTime</a> is negative, throw a <a data-cite="WEBIDL#syntaxerror"><code>SyntaxError</code></a>.</li>
+                  <li>If <var>markOptions</var>'s <a>startTime</a> is negative, throw a <a data-cite="WEBIDL#exceptiondef-rangeerror"><code>RangeError</code></a>.</li>

```

Comment:
Usually this would be a TypeError in web APIs (e.g. that's what WebIDL uses for [EnforceRange] violations).

Reply:
I'm curious - is there a document I could read to better outline which errors should be thrown in which occasions? https://heycam.github.io/webidl/#idl-exceptions is not super descriptive when it comes to when each one of them should be used

Reply:
I don't know that we have anything great.  :(